src/main/groovy/com/twcable/grabbit/client/servlets/GrabbitJobServlet.groovy

```diff
@@ -147,6 +147,28 @@ class GrabbitJobServlet extends SlingAllMethodsServlet {
         response.writer.write(new JsonBuilder(jobExecutionIds).toString())
     }
 
+    @Override
+    protected void doPost(SlingHttpServletRequest request, SlingHttpServletResponse response)  {
+        String jobExecutionId = request.getParameter("jobId") ?: ""
+        if(!jobExecutionId.isLong()) {
+            log.warn "Parameter ${jobExecutionId} 'jobId' is invalid"
+            response.status = SC_BAD_REQUEST
+            response.writer.write("Parameter 'jobId' is invalid")
+            return
+        }
+        try {
```

Comment:
@viveksachdeva I'm not sure what you mean. All jobs, including ones being run are backed by the grabbit repository. 

I think we should have the distinction...and at the very least do away with catching "Exception" for reasons I've mentioned before. It will just save us from having to clean this up in the future, because the Checker framework will complain about it..